source/includes/connection-snippets/x509-snippets/perl-connection.pl

```diff
@@ -0,0 +1,14 @@
+# begin x509 connection
+use IO::Socket::SSL;
+use MongoDB;
+
+my $client = MongoDB::MongoClient->new(
+    host => "mongodb+srv://<cluster-url>",
```

Comment:
Is this missing the X509 subject? Connection strings for other language include that. /cc @xdg

Reply:
For 3.4+, we shouldn't need to give the subject as username, as it can be picked out of the ticket.  And drivers can (as the Perl driver does) pick it out of the ticket and send it for earlier servers.  See https://github.com/mongodb/specifications/blob/master/source/auth/auth.rst#mongodb-x509

Reply:
Ah, in that case it was probably a mistake for other driver examples in this PR to include a subject. Thanks for the clarification!